src/OSPSuite.Core/Serialization/Xml/ColumnXmlSerializer.cs

```diff
@@ -12,14 +13,23 @@ public override void PerformMapping()
          Map(x => x.LloqColumn);
          Map(x => x.Name);
          Map(x => x.Unit);
+         Map(x => x.Dimension);
       }
 
       protected override void TypedDeserialize(Column objectToDeserialize, XElement outputToDeserialize, SerializationContext context)
       {
          base.TypedDeserialize(objectToDeserialize, outputToDeserialize, context);
-         
+
          if (objectToDeserialize.ErrorStdDev == Constants.STD_DEV_GEOMETRIC)
+         {
             objectToDeserialize.Unit = new UnitDescription();
+            objectToDeserialize.Dimension = Constants.Dimension.NO_DIMENSION;
+         }
+
+         if (!objectToDeserialize.Unit.ColumnName.IsNullOrEmpty())
```

Comment:
Can Unit be NULL here?

Reply:
It could be empty. In case the units are set manually there is no ColumnName. I would not expect null but just to be on the safe side with code evolving in the future I rather keep IsNullOrEmpy() and not != null.